widget/playground/src/containers/StyleLayout/StyleLayout.General.tsx

```diff
@@ -43,7 +42,7 @@ export function General() {
   const secondaryBorderRadius =
     useConfigStore.use.config().theme?.secondaryBorderRadius;
   const fontFamily =
-    useConfigStore.use.config().theme?.fontFamily || FONTS[0].value;
+    useConfigStore.use.config().theme?.fontFamily || SUPPORTED_FONTS[0].value;
```

Comment:
You don't need to export `SUPPORTED_FONTS` from `embedded`, Please make a constant called `DEFAULT_FONT` in playground itself and use it here.